src/odbc_connection.cpp

```diff
@@ -2931,8 +3077,14 @@ fetch_and_store
         // Copy the data over if the row status array indicates success
         if
         (
-          data->row_status_array[row_index] == SQL_ROW_SUCCESS ||
-          data->row_status_array[row_index] == SQL_ROW_SUCCESS_WITH_INFO
+          data->simple_binding == true ||
```

Comment:
```suggestion
          data->simple_binding ||
```
No point in comparing a boolean to anything, since it already is a boolean